src/Campaigns/Migrations/Tables/CreateCampaignsTable.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Give\Campaigns\Migrations\Tables;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ * Creates give_campaigns table
+ */
+class CreateCampaignsTable extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public static function id(): string
+    {
+        return 'give-campaigns-create-give-core-campaigns-table';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function title(): string
+    {
+        return 'Create give_campaigns table from core';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function timestamp(): string
+    {
+        return strtotime('2024-08-26 00:00:00');
+    }
+
+    /**
+     * @inheritDoc
+     * @throws DatabaseMigrationException
+     */
+    public function run()
+    {
+        global $wpdb;
+
+        $table = $wpdb->prefix . 'give_campaigns';
```

Comment:
In P2P we registered the table name in the Service Provider to avoid the need for prefixing elsewhere.

```php
public function register() {
  global $wpdb;
  $wpdb->give_campaigns = "{$wpdb->prefix}give_campaigns";
}
```

Reply:
Yes, but that will work only if P2P is installed on the site.

Reply:
Ah, yes, I mean that we should move over the table name registration to core also.

Reply:
I commented on the migration, but the definition would need to be added to the Service Provider.